tests/garage/tf/envs/test_gym_base.py

```diff
@@ -19,6 +19,9 @@ def test_all_gym_envs(self, spec):
         if spec._env_name.startswith('CarRacing'):
             pytest.skip(
                 'CarRacing-* envs bundled in atari-py 0.2.x don\'t load')
+        if spec._env_name.startswith('KellyCoinflip'):
+            pytest.skip(
+                'KellyCoinflip env has tuple observation, not np.array')
```

Comment:
This explanation isn't *quite* right. The problem isn't that it has tuple observations exactly, it's that those observations aren't contained in the `observation_space`, and also aren't a `Box` (which we have special logic for when it exceeds the `observation_space`).